publish.ps1

```diff
@@ -0,0 +1,46 @@
+# **************************************************************************
+#  Copyright (c) Cloud Native Foundation.
+#  SPDX-License-Identifier: Apache-2.0
+# **************************************************************************
+
+<#
+    .SYNOPSIS
+    Publish the CloudEvents.Sdk module to PSGallery
+
+    .PARAMETER NuGetApiKey
+    PowerShell Gallery API Key to be used to publish the module
+#>
+
+param(
+    [Parameter(Mandatory = $true)]
+    [string]
+    $NuGetApiKey
+)
+
+$moduleName = 'CloudEvents.Sdk'
+$OutputDir = Join-Path $PSScriptRoot $moduleName
+
+if (-not (Test-Path $OutputDir)) {
+    New-Item -ItemType Directory -Path $OutputDir | Out-Null
+}
+
+# Build and Test before publish
```

Comment:
Can we extract this (e.g. as we do in PR runs) and use build/test as a separate step in the workflow instead of combining it into a larger step?

Reply:
sure.